Add bang operator to operators table (#6558)
godotengine/godot-docs: tutorials/scripting/gdscript/gdscript_basics.rst

+--------------------------------------------------------------------------------+-------------------------------------------+
| ``in``                                                                         | Inclusion checker (when used with         |
|                                                                                | control flow keywords or in a             |
|                                                                                | standalone expression)                    |
|                                                                                |                                           |
|                                                                                | Content iterator (when used with the      |
|                                                                                | for_ keyword)                             |
+--------------------------------------------------------------------------------+-------------------------------------------+

> | ``not`` ``!``                                                                  | Boolean NOT and it's                      |
> |                                                                                | :ref:`aliase<boolean_operators>`          |

Reviewer: ```suggestion
| ``not`` ``!``                                                                  | Boolean NOT and its                       |
|                                                                                | :ref:`aliases<boolean_operators>`          |
```

> |                                                                                | :ref:`aliases<boolean_operators>`          |

Reviewer: ```suggestion
|                                                                                | :ref:`aliases<boolean_operators>`         |
```

+--------------------------------------------------------------------------------+-------------------------------------------+

> | ``and`` ``&&``                                                                 | Boolean AND and it's                      |
> |                                                                                | :ref:`aliase<boolean_operators>`          |

Reviewer: ```suggestion
| ``and`` ``&&``                                                                 | Boolean AND and its                       |
|                                                                                | :ref:`aliases<boolean_operators>`         |
```

+--------------------------------------------------------------------------------+-------------------------------------------+

> | ``or`` ``||``                                                                  | Boolean OR and it's                       |
> |                                                                                | :ref:`aliase<boolean_operators>`          |

Reviewer: ```suggestion
| ``or`` ``||``                                                                  | Boolean OR and its                        |
|                                                                                | :ref:`aliases<boolean_operators>`         |
```
